pkg/libursa/ursa/ursa.go

```diff
@@ -119,6 +119,28 @@ func CredentialPublicKeyFromJSON(jsn string) (unsafe.Pointer, error) {
 	return handle, nil
 }
 
+//NewCredentialDef creates and returns credential definition (public and private keys, correctness proof) entities
+func NewCredentialDef(schema, nonSchema unsafe.Pointer, revocation bool) (pubKey unsafe.Pointer, privKey unsafe.Pointer,
+	proof unsafe.Pointer, err error) {
+
+	rev := C.bool(revocation)
+	defer C.free(unsafe.Pointer(&rev))
+
+	var credpub, credpriv, credproof unsafe.Pointer
+	defer func() {
+		C.ursa_cl_credential_public_key_free(credpub)
+		C.ursa_cl_credential_private_key_free(credpriv)
+		C.ursa_cl_blinded_credential_secrets_correctness_proof_free(credproof)
+	}()
+
```

Comment:
You can't free these handles when you are returning them from the method.  The caller of this func will get panics